### PR TITLE
Update Kubernetes examples

### PR DESCRIPTION
**Pull request by Starefossen**
This PR:

* [x] adds `hostNetwork: true` to Kubernetes Deployment. This is required in order to bind directly to host ports.
* [x] removes duplicated Kubernetes examples so we only have one source of up to date examples for Kubernetes.

### CODE REVIEW

**Comment by ekozan** [comment]
it's not a good practice for the safety and it's won't work on some case with "hostNetwork: true"

the best is to use service with externalIP 
or hostPort on de deployment when they correct the actual ""bug"" with this on the cni plugins

**Comment by Starefossen** [comment]
Yeah, that is true @ekozan.

Either that or remove the hostPort all together as it does not work as it is on `1.4.0`.

**Comment by ekozan** [comment]
https://github.com/kubernetes/kubernetes/issues/31307 it's will be fix soon .... i hope :D

**Comment by Starefossen** [comment]
This is great news 😄

**Comment by emilevauge** [comment]
Ping @errm :)

File: docs/toml.md [hunk 1]
```diff
@@ -944,7 +944,7 @@ Annotations can be used on containers to override default behaviour for the whol
 
 - `traefik.frontend.rule.type: PathPrefixStrip`: override the default frontend rule type (Default: `PathPrefix`).
 
-You can find here an example [ingress](https://raw.githubusercontent.com/containous/traefik/master/examples/k8s.ingress.yaml) and [replication controller](https://raw.githubusercontent.com/containous/traefik/master/examples/k8s.rc.yaml).
+You can find here an example [ingress](https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/checkes-ingress.yaml) and [replication controller](https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik.yaml).
```
Comment (emilevauge) [inline docs/toml.md]: You mean `cheese-ingress.yaml` right ?

File: examples/k8s/traefik.yaml [hunk 2]
```diff
@@ -16,11 +16,12 @@ spec:
       labels:
         k8s-app: traefik-ingress-lb
         name: traefik-ingress-lb
-        version: v1.0.0
+        version: v1.0.3
```
Comment (emilevauge) [inline examples/k8s/traefik.yaml]: I think it would be even better to go on `v1.1.0` ;)

**Comment by Starefossen** [comment]
Updated and rebased on `origin/v1.1` 😄

**Comment by ekozan** [comment]
Starefossen do you have an cluster with and CNI like Weave on it ? and this hack work ? 

Because on my cluster it's just dont work like I said $_$

**Comment by Starefossen** [comment]
@ekozan yes, I have a cluster where this technique is working running `kubernetes-cni` and `Weave` on Oracle Enterprise Linux 7 set up by `kubeadm`.

**Comment by Starefossen** [comment]
@ekozan if you joine the Træffik Slack Channel https://traefik.herokuapp.com/ I can help you debug this. Will be online for another hour.

**Pull request merged**